web-ui/vite.config.ts

```diff
@@ -1,23 +1,23 @@
-import { defineConfig } from "vitest/config"
-import react from "@vitejs/plugin-react-swc"
+import { defineConfig } from 'vitest/config';
+import react from '@vitejs/plugin-react-swc';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  base: "/",
+  base: '/',
   plugins: [react()],
   test: {
     globals: true,
-    environment: "jsdom",
-    setupFiles: "./src/setupTests.js",
-    css: true,
-    reporters: ["verbose"],
+    environment: 'jsdom',
+    setupFiles: './src/setupTests.js',
+    css: false,
```

Comment:
Setting `css` to `false` is the only real change made in this file. Before this change we were getting the error "Could not parse CSS stylesheet" for any CSS files imported by a component that use nested CSS rules. It seems we don't really need to parse the CSS in order for all our existing tests to pass. I verified that.